### PR TITLE
Translated Privacy.md and Tos.md to french

### PR DESCRIPTION
**Pull request by louvenarde**

### CODE REVIEW

File: templates/views/privacy.md [hunk 1]
```diff
@@ -1,62 +1,69 @@
-# Forged Alliance Forever Privacy Statement
+# Déclaration de confidentialité de Forged Alliance Forever
 
-Effective date: _May 25, 2018_
+Merci d'avoir joué à Supreme Commander : Forged Alliance dans notre communauté. Nous voulons faire en sorte qu'il soit facile pour vous de conserver vos informations personnelles et contrôler votre identité virtuelle, et dans ce but, ce document présente dans quelle mesure nous collectons ces données et de quelle façon nous les traitons
```
Comment (Sheeo) [inline templates/views/privacy.md]: Why's the effective date not mentioned?

File: templates/views/privacy.md [hunk 2]
```diff
@@ -1,62 +1,69 @@
...
+
+Nous prenons toutes les mesures nécéssaires pour protéger vos Informations Utilisateur Personnelles, par l'utilisation des standards industriels pour la sécurité des technologies. En cas de brèche de données, nous vous en informerons rapidement.
+
+## Gestion des plaintes
+
+Si vous avez des plaintes concernant la façon dont FAForever gère vos Informations Utilisateur Personnelles, veuillez nous le faire savoir immédiatement à admin@faforever.com
+
+## Modifications de notre déclaration de confidentialité
+
+La plupart des changements de ce document ont de bonnes chances d'être très mineurs, mais s'il devaient se produire, vous en serez informés et disposerez d'au moins 30 jours de délai avant l'application desdits changements.
```
Comment (Sheeo) [inline templates/views/privacy.md:66]: The privacy statement should end with a section clarifying that the English version is the "authority" should there be any dispute in wording/loss of translation.

File: templates/views/tos.md [hunk 3]
```diff
@@ -1,179 +1,159 @@
-# Forged Alliance Forever Terms of Service
+# Conditions d'utilisations de Forged Alliance Forever
 
-Effective date: _May 25, 2018_
+Date d'application: _25 Mai 2018_
 
-## Short Version
+## Version courte
 
-Thanks for creating an account with Forged Alliance Forever. This document outlines our Terms of Service. In short:
```
Comment (Sheeo) [inline templates/views/tos.md:7]: The ToS should start with a disclaimer that the legally binding document is the English version, and a link to it directly.

**Review by Sheeo — CHANGES_REQUESTED** [review]

**Comment by Sheeo** [comment]
Thanks a lot so far, if you don't mind amending with my comments I'll get right on integrating the translation in full.

File: templates/views/privacy.md (hunk 2 above)
Comment (Sheeo) [reply templates/views/privacy.md:66]: See the [german translation](https://github.com/FAForever/website/pull/143/files#diff-6f6adf454a28ca7deedb2bc3174e7189R73) for an example :)

File: templates/views/privacy.md (hunk 1 above)
Comment (louvenarde) [reply templates/views/privacy.md]: It got automatically deleted when importing in Poe editor, you're right. Will fix.

File: templates/views/privacy.md (hunk 2 above)
Comment (louvenarde) [reply templates/views/privacy.md:66]: Okay, will do

File: templates/views/tos.md (hunk 3 above)
Comment (louvenarde) [reply templates/views/tos.md:7]: Okay, will do

**Pull request closed without merging**